### PR TITLE
pingall.coffee: add cronjob to pingall

### PR DESCRIPTION
**Pull request by sbourdelin**

### CODE REVIEW

File: src/pingall.coffee [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 # Commands:
 #   hubot pingall <message> - ping all users on the channel with the optional message
+#   hubot add pingall cron <crontab pattern> <"message"> - add a cron job to ping all users on the channel with message
 #
 # Author:
 #   ephelip
+#   sebastien.bourdelin@savoirfairelinux.com
 
 util = require 'util'
+HubotCron = require 'hubot-cronjob'
```
Comment (ephelip) [inline src/pingall.coffee:13]: you should add this dependency to package.json

File: src/pingall.coffee [hunk 2]
```diff
@@ -19,3 +22,25 @@ module.exports = (robot) ->
...
+        # format cronjob
+        timezone = 'America/Montreal'
+        pattern = context.match[1]
+        message = context.match[2]
+
+        pingallCron = () ->
+            context.send '[' + usersStr + ']' + " " + message
+        
+        try
+            new HubotCron pattern, timezone, pingallCron
```
Comment (ephelip) [inline src/pingall.coffee:43]: Does hubot sends the cron messages only on the chan where he received the command ?

**Review by ephelip — CHANGES_REQUESTED** [review]

File: src/pingall.coffee (hunk 1 above)
Comment (sbourdelin) [reply src/pingall.coffee:13]: ok! thx

File: src/pingall.coffee (hunk 2 above)
Comment (sbourdelin) [reply src/pingall.coffee:43]: Yes we keep the context

**Review by ephelip — APPROVED** [review]
great ! Thanks :)

**Pull request merged**